test/e2e/storage/pd.go

```diff
@@ -449,7 +449,7 @@ var _ = SIGDescribe("Pod Disks", func() {
 		// Verify that disk shows up in node 0's volumeInUse list
 		framework.ExpectNoError(waitForPDInVolumesInUse(nodeClient, diskName, host0Name, nodeStatusTimeout, true /* should exist*/))
 
-		output, err := exec.Command("gcloud", "compute", "instances", "list").CombinedOutput()
+		output, err := exec.Command("gcloud", "compute", "instances", "list", "--project="+framework.TestContext.CloudConfig.ProjectID).CombinedOutput()
```

Comment:
I agree we should switch all gcloud CLI calls to API. When the E2E tests were first written the client API code wasn't integrated/initialized so everyone just used direct CLI gcloud commands. I added the integration (https://github.com/kubernetes/kubernetes/pull/17747), but never got around to switching everything over.